pywt/_multidim.py

```diff
@@ -107,6 +82,10 @@ def idwt2(coeffs, wavelet, mode='symmetric'):
         Wavelet to use
     mode : str, optional
         Signal extension mode, see Modes (default: 'symmetric')
+    axes : 2-tuple of ints, optional
+        Axes over which to compute the IDWT. Repeated elements mean the IDWT
+        will be performed multiple times along these axes. A value of `None`
```

Comment:
I thought `2-tuple of ints` was clear that a tuple containing 2 integers is required. I can spell it out again in the body if necessary.